packages/aila/src/protocol/schema.ts

```diff
@@ -71,6 +72,25 @@ export type MisconceptionsOptional = z.infer<
 
 // ********** QUIZ **********
 
+// TODO: GCLOMAX: Relax this constraint to allow for more than one answer.
```

Comment:
Are there any such constraints on the data you're working with? e.g. a max number of answers/distractors?